src/content/docs/configuration/sharing.mdx

```diff
@@ -93,4 +93,9 @@ pull_request_rules:
 
   - Values in the `default` key will be merged and remote default values will
     apply to local configuration.
+
+  - If there is rule in any of the `pull_request_rules`, `queue_rules`, `priority_rules`, `partition_rules`
+    or `merge_protections` section with the same name in both the extended config and the
```

Comment:
what do you think of introducing the words `parent config` and `child config` to make the concept easier to grasp?

Reply:
Good idea, made a modification, let me know if that sounds good know